### DISCUSSION TITLE
Automatic Failure Detection?

### DISCUSSION THREAD

**Discussion by Kokunekosan** [lllyasviel/Fooocus#3355] [Ideas]
Would it be possible to add a mechanism like this to Fooocus? If, say, the image loses all noise, or gets stuck on the same pattern for more than, like, 5 samplings, either end that generation attempt or restart?

**Comment by mashb1t** [comment]
Please elaborate, i don't get what you mean.

**Comment by Kokunekosan** [comment]
![2024-07-25_02-56-49_1502](https://github.com/user-attachments/assets/64e64118-94f3-4b82-9bc0-c95e081bcae2)
this, or an entirely blank image (as in, all one color)

**Reply by mashb1t** [reply]
What's your prompt and sampler etc, did you define any subject? I still don't get your intention or feature request.

**Reply by Kokunekosan** [reply]
I was just curious if it would be possible to add a tool that would automatically skip failing renders before they run to completion. I don't _mind_ manually deleting failures, but if fooocus could automatically skip a bad seed for that prompt or whatever, it would be... neat? I've only been playing with your cool program for a couple of weeks, so there are definitely things I'm doing wrong.

**Reply by Kokunekosan** [reply]
and i've got embed metadata on, so if you want to get the prompt and all that, it's in the image

**Reply by mashb1t** [reply]
No, tis is a won't do. Please consider writing more detailed prompt to prevent generating images as the one posted by you above (depending what your prompt was this might have also been intended, i don't know). Guide for writing prompts: https://stable-diffusion-art.com/prompt-guide/

**Reply by mashb1t** [reply]
Just checked out your metadata. Please be aware of that for Pony all LoRAs have to be compatible with it, not SDXL LoRAs.
Also **you're using 20 (!!) LoRAs**, which is a crazy high amount. Please consider using max. 4-5 as weight adjustments will conflict with such a high amount.

**Comment by Kokunekosan** [comment]
I've seen images get stuck in this state anywhere from sample 5 to 30 and then just keep rendering the same frame over and over, or the noise just fades away into a solid color
